code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -337,3 +338,13 @@ var/list/mob_hat_cache = list()
 /mob/living/silicon/robot/drone/construction/updatename()
 	real_name = "construction drone ([rand(100,999)])"
 	name = real_name
+
+/mob/living/silicon/robot/drone
+	can_enter_vent_with = list(
```

Comment:
You might as well make them able to ventcrawl with any /obj, so you don't need to keep de-equipping your modules to crawl around. You can just re-equip them once you're in the pipe, so it's just an annoyance.

Reply:
They were implemented with them stowed and that's how they were mentioned in the staff meeting, so I'll leave it unless @Anewbe says otherwise.

Reply:
Don't care how it works, just want Drones with ventcrawl.

Reply:
So, if I just make the can_ventcrawl into /obj that would allow them to ventcrawl with anything, yes? Or no?

Reply:
/obj or /obj/item would allow it, yes.

Reply:
doned